models/intel/yolo-v2-ava-sparse-35-0001/description/yolo-tiny-v2-ava-sparse-30-0001.md

```diff
@@ -0,0 +1,48 @@
+# yolo-v2-ava-sparse-35-0001
```

Comment:
The filename doesn't match the model name.

Reply:
seems fixed